core/src/Cabana_Experimental_NeighborList.hpp

```diff
@@ -165,17 +164,39 @@ struct NeighborDiscriminatorCallback2D_FirstPass
     }
 };
 
+// Preallocate and attempt fill in the first pass
+template <typename Counts, typename Neighbors, typename Tag>
+struct NeighborDiscriminatorCallback2D_FirstPass_BufferOptimization
+{
+    Counts counts;
+    Neighbors neighbors;
+    using tag = ArborX::Details::InlineCallbackTag;
```

Comment:
I thought we got rid of these?

Reply:
It is being ignored.  But yes can safely be removed.